constellations/onboarding/guide.md

```diff
@@ -19,7 +19,22 @@ Let's get started!
 
 ***Warning for Mac users*** At this time, Docker-Desktop is not supported for Macs older than 2010 or OS versions earlier than 10.13. If you have an older Mac OS, please refer to the [alternative guide](./guide_alternative.md).
 
-***Warning for Windows users*** At this time, Docker-Desktop is only supported for Windows 10 Pro/Enterprise. Docker is not supporting Windows 10 Home until version 2004 (coming in May/June 2020). If Docker-Desktop cannot be installed, please refer to the [alternative guide](./guide_alternative.md).
+***Warning for Windows users*** We will be downloading Windows 10, version 2004. This will allow us to download docker-desktop. First you need to download MySQL and GoLang. [MySQL download for Mac and Windows Directions](https://github.com/ahsu1230/mathnavigatorSite/blob/master/resources/install_mysql/guide.md). 
+[Go Download](https://golang.org/).
+
+### Windows 10, version 2004 download 
+You can google this download or use the link here [Windows Support Website](https://support.microsoft.com/en-us/help/4027667/windows-10-update?fbclid=IwAR01JBDzt270x4OaxsNkYlBjG4-sVO21NfaEeYbHCYhcJnYilbylwXnMBdQ). This link will direct you to the Windows Support Website. 
+Here when you click the blue Check for Windows Update button at the bottom, this button will direct you to a system update on your windows computer. 
+Hopefully, you will see an update for Windows 10, version 2004. Please install and go through the instructions.
+
+#### Docker-Desktop Download for Windows
+Once your computer has restarted, please go online and download Docker-Desktop. There may be a problem when you first try and open the software. Docker-Desktop should give an error that you don't have a certain file. 
+When you open the link, Click on the link that says “Please download the latest WSL2 Linux kernel update package for x64 machines.” 
+This will then allow you to open the software and do the tutorial. After this is done, you will now be able to now follow the rest of this github guide. 
+
+#### Windows download for the rest of the Github Guide
+When you follow the commands in the rest of this github guide, make sure you run the command prompt on Windows as an administrator. This will allow you to open and stop MySQL from running so you can download these files. 
```

Comment:
Since we're moving the instructions to another file, you should provide a link back to this guide and specifically to the header of where they should be left off to like: ./onboarding/guide.md#starting-gemini-admin